modules/retail/pages/retail-install-setup.adoc

```diff
@@ -19,8 +19,8 @@ Each procedure is detailed in this section.
 
 == Install the build host and register it to {susemgr}
 
-Your build host should be a Salt minion, running {sls}{nbsp}12 SP3.
-For instructions to install Salt minion see https://www.suse.com/documentation/suse-manager-3/3.2/susemanager-getting-started/html/book.suma.getting-started/preparing.and.registering.clients.html
+Your build host should be a Salt minion, running {sls}{nbsp}15 SP1.
```

Comment:
This is not correct

It should be 12 SP3 or SP4 for sles12 images (SP4 recommended)
and 11 SP4 x86_64 for sles11 images

Reply:
ok, I'll change that accordingly.